deployment/ansible/roles/cac-tripplanner.app/tasks/jslibs.yml

```diff
@@ -11,7 +11,7 @@
        state=present
 
 - name: Install NPM dependencies
-  npm: path="{{ root_src_dir }}" state=present
+  command: npm install chdir="{{ root_src_dir }}"
```

Comment:
Yes -- without this one of the gulp dependencies would fail to get installed, even after deleting node_modules; further details: https://github.com/azavea/cac-tripplanner/pull/17#discussion_r23372922